pages/common/od.md

```diff
@@ -0,0 +1,25 @@
+# od
+
+> Display file contents in hexadecimal, octal or printable character format optionally along with
```

Comment:
Please split the sentence before "optionally" and add the second sentence entirely in the second line.